hedera-mirror-rosetta/cmd/main.go

```diff
@@ -20,11 +21,14 @@ import (
 func NewBlockchainRouter(network *types.NetworkIdentifier, asserter *asserter.Asserter, version *types.Version, dbClient *gorm.DB) http.Handler {
 	blockRepo := block.NewBlockRepository(dbClient)
 	transactionRepo := transaction.NewTransactionRepository(dbClient)
+	accountRepo := account.NewAccountRepository(dbClient)
+
+	commons := services.NewCommons(blockRepo, transactionRepo)
 
 	networkAPIService := services.NewNetworkAPIService(network, version, blockRepo, transactionRepo)
```

Comment:
A good idea here is to use commons as well

Reply:
yes, you are right